pkg/karmadactl/version.go

```diff
@@ -0,0 +1,33 @@
+package karmadactl
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"github.com/karmada-io/karmada/pkg/version"
+)
+
+var (
+	versionLong = `Version prints the version info of this command.`
+
+	versionExample = `
+		# Print karmadactl command version
+		karmadactl version`
```

Comment:
Too much indents here:
```bash
# karmadactl version --help 
Version prints the version info of this command.

Usage:
  karmadactl version [flags]

Examples:

		# Print karmadactl command version
		karmadactl version

```

How about this?
```suggestion
	versionExample = `  # Print karmadactl command version
  karmadactl version`
```
That will makes:
```
# ./karmadactl version --help 
Version prints the version info of this command.

Usage:
  karmadactl version [flags]

Examples:
  # Print karmadactl command version
  karmadactl version
```

Reply:
> Too much indents here:
> 
> ```shell
> # karmadactl version --help 
> Version prints the version info of this command.
> 
> Usage:
>   karmadactl version [flags]
> 
> Examples:
> 
> 		# Print karmadactl command version
> 		karmadactl version
> ```
> 
> How about this?
> 
> That will makes:
> 
> ```
> # ./karmadactl version --help 
> Version prints the version info of this command.
> 
> Usage:
>   karmadactl version [flags]
> 
> Examples:
>   # Print karmadactl command version
>   karmadactl version
> ```

Yes, I have noticed it too, thanks.

Reply:
> Too much indents here:
> 
> ```shell
> # karmadactl version --help 
> Version prints the version info of this command.
> 
> Usage:
>   karmadactl version [flags]
> 
> Examples:
> 
> 		# Print karmadactl command version
> 		karmadactl version
> ```
> 
> How about this?
> 
> That will makes:
> 
> ```
> # ./karmadactl version --help 
> Version prints the version info of this command.
> 
> Usage:
>   karmadactl version [flags]
> 
> Examples:
>   # Print karmadactl command version
>   karmadactl version
> ```

Yes, I have noticed it too, thanks.